CMakeLists.txt

```diff
@@ -97,6 +97,7 @@ cvc5_option(ENABLE_COMP_INC_TRACK
             "Enable optimizations for incremental SMT-COMP tracks")
 cvc5_option(ENABLE_DEBUG_SYMBOLS  "Enable debug symbols")
 cvc5_option(ENABLE_MUZZLE         "Suppress ALL non-result output")
+cvc5_option(ENABLE_SAFE_OPTS      "Enable safe-options by default")
```

Comment:
Can you rename this `ENABLE_SAFE`?  with description `"Enable safe mode"`.

We haven't fully settled on the naming here, but `safe-options` is conceptually only part of what this mode will do.

Reply:
Done